erizo/src/erizo/rtp/QualityFilterHandler.h

```diff
@@ -64,6 +64,9 @@ class QualityFilterHandler: public Handler, public std::enable_shared_from_this<
   time_point time_change_started_;
   int picture_id_offset_;
   int last_picture_id_sent_;
+  clock::time_point last_sent_packet = clock::now();
+  clock::time_point cooldown_target;
+  bool cooldown = false;
```

Comment:
consider using `cooldown_`

Reply:
also, I prefer initializing all things in the same place, so use the constructor to initialize this variable please.